R/center.R

```diff
@@ -32,11 +32,16 @@
 #'  argument of `prep.recipe`. `bake.recipe` then applies
 #'  the centering to new data sets using these means.
 #'
-#'  # Tidying
+#' # Tidying
 #'
-#'  When you [`tidy()`][tidy.recipe()] this step, a tibble with columns
-#'  `terms` (the selectors or variables selected) and `value` (the means)
-#'  is returned.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble is returned with 3
+#' columns `terms`, `value` , and `id`:
```

Comment:
It looks like a search + replace might have resulted in an additional space before the oxford commas. 

```suggestion
#' columns `terms`, `value`, and `id`:
```